app/routes.js

```diff
@@ -26,13 +26,13 @@ const secEvents = require('./middleware/secEvents');
 module.exports = (app, passport, mongoose) => {
 
     // setup resume repo password for Companies Only
-    const basic = auth.basic({
-        realm: 'Sponsoring companies only.',
-        file: __dirname + '/users.htpasswd'
-      });
+    // const basic = auth.basic({
+    //     realm: 'Sponsoring companies only.',
+    //     file: __dirname + '/users.htpasswd'
+    //   });
 
     // get the page for all student repo's
-    app.get('/repo', auth.connect(basic), (req, res) => {
+    app.get('/repo', (req, res) => {
```

Comment:
Doing PR's, we can revisit this later. Currently doing manual dumps. I want the basic functionality